documentation/src/main/asciidoc/topics/hotrod_protocol.adoc

```diff
@@ -2034,13 +2034,26 @@ Here are the current values for each flag: +
 .Infinispan versions
 TIP: This version of the protocol is implemented since Infinispan 9.2
 
-This Hot Rod protocol version adds support for transaction operations.
-It includes 3 new operations:
+This Hot Rod protocol version adds support for transaction operations and multimap caches.
+
+.New operations for transactions:
 
 * Prepare, with the transaction write set (i.e. modified keys), it tries to prepare and validate the transaction in the server.
 * Commit, commits a prepared transaction.
 * Rollback, rollbacks a prepared transaction.
 
+. New operations to manipulate multimap caches.
+
+* request 0x67 - response 0x68 = Get a key and retrieve multiple values
+* request 0x69 - response 0x6A = Get a key with its metadata and retrieve multiple values
+* request 0x6B - response 0x6C = Put a key/value entry in the cache
+* request 0x6D - response 0x6E = Remove a key
+* request 0x6F - response 0x70 = Remove an entry
+* request 0x71 - response 0x72 = Get cache size
```

Comment:
@karesti Is "Get cache size" correct here? Or is it the size of a key/entry?

Reply:
size of the cache